src/sentry/api/serializers/models/projectcodeowners.py

```diff
@@ -12,7 +12,9 @@ def serialize(self, obj, attrs, user):
             "codeMappingId": str(obj.repository_project_path_config_id),
             "provider": "unknown",
         }
-        if obj.organization_integration:
-            data["provider"] = obj.organization_integration.integration.provider
+        if obj.repository_project_path_config.organization_integration:
```

Comment:
A bit concerned that our tests didn't catch this. I think we need a test using this serializer with a ProjectCodeOwners that has an `organization_integration.

Reply:
@wedamija we have tests for this here: https://github.com/getsentry/sentry/blob/10a42806d162e18c4bde52248db7a4bf1abebcd4/tests/sentry/api/endpoints/test_project_codeowners.py#L53-L78

I tried it locally with the previous version, and they failed locally. Not sure why CI didn't fail...